services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/model/FileUpload.java

```diff
@@ -27,13 +29,19 @@
 public interface FileUpload extends ObjectTransfer {
 
     /**
-     * Pause the current upload operation and return the information that can
+     * Pauses the current upload operation and return the information that can
      * be used to resume the upload at a later time.
      * <p>
      * The information object is serializable for persistent storage until it should be resumed.
      * See {@link ResumableFileUpload} for supported formats.
+     * 
+     * <p>
+     * Currently, it's only supported if the underlying {@link S3AsyncClient} is CRT-based created via
+     * {@link S3AsyncClient#crtBuilder()} or {@link S3AsyncClient#crtCreate()}.
```

Comment:
...if the underlying {@link S3AsyncClient} is CRT-based created via {@link S3AsyncClient#crtBuilder()} or {@link S3AsyncClient#crtCreate()}

Right now it reads as if there may be more methods than these two that can create an CRT based client so you should only use those, but I am assuming that you have to use a CRT based client and that linking the methods is a helpful hint. 

Maybe parentheses around methods? "...if the underlying {@link S3AsyncClient} is CRT-based (created via {@link S3AsyncClient#crtBuilder()} or {@link S3AsyncClient#crtCreate()})"? Or just use i.e. as below.